### ISSUE TITLE
Typing QQ from normal mode triggers Ex mode instead.

### ISSUE THREAD

**Issue by edthedev** [nicwest/QQ.vim#19]
To duplicate:
Vim version 7.3.547
Remove any nnoremap Q lines from the Vimrc.
(May be present, since disabling Ex mode is common... http://www.bestofvim.com/tip/leave-ex-mode-good/)
Relaunch Vim
Press QQ.

**Comment by nicwest** [comment]
I'll have a look and see if there is anything that can be done.

In the meantime however there is `g:QQ_map_prefix` which by default is `Q` that you can set to whatever you like.   
I'm not sure if I got round to fully implementing it, so if anything breaks let me know!

**Comment by nicwest** [comment]
So I had a play around, and the closest version I could get to yours was 7.3.429. I can't really see the issue. I might not be understanding the issue?  

here is the vagrant file for the set up i was using to test:   
https://gist.github.com/a04069b3308a4c42d581

```
vagrant up
vagrant ssh
vim
QQ
```

going to close this for now.

**Comment by edthedev** [comment]
I'll try the Vagrant box and then see if I can spot the difference.

The server I have that version of Vim on is just a Debian 7, server. I'll
see if updating Vim to the latest available makes any difference and report
back, as well.

**Comment by nicwest** [comment]
ok cool so I changed the vagrant set up to debian 7.4 and that got me the right version of vim:

https://gist.github.com/5ca1af7dbc360d9d4ebb

Again I'm seeing no issues opening the request window. Is it possible that the plugin was not installed correctly on your machine?

**Comment by edthedev** [comment]
I installed through Vundle, which appears to be the issue.
I've never seen this before, but it happened twice, once on my Debian server and once on my Macbook.

After adding Plugin 'nicwest/QQ.vim' to the appropriate place in the .vimrc, I ran :BundleInstall

But when I try to confirm the plugin install:
ls ~/.vim/bundle

QQ.vim is missing.

Re-running 'vim +BundleInstall', or opening vim and running :BundleInstall solves the issue on both hosts, the MacBook and Debian.
